### DISCUSSION TITLE
Has the function of hiding a specific dispatcher disappeared?

### DISCUSSION THREAD

**Discussion by DOONG2** [reduxjs/redux-devtools#1060] [Q&A]
![image](https://user-images.githubusercontent.com/35028305/151275205-d5763039-743e-4aed-934e-1c01f35dac24.png)

**Comment by Methuselah96** [comment]
I'm not sure what you mean. Does it not let you type actions in the box?

**Reply by DOONG2** [reply]
I'm sorry. I didn't explain enough. 🙏
I think the function of hiding a specific dispatcher disappeared after the update.
Is there a function to filter a specific dispatcher in this version?

**Reply by DOONG2** [reply]
The picture above is the previous version of the UI.
